examples/platform/esp32/common/Esp32AppServer.cpp

```diff
@@ -55,13 +46,6 @@ void Esp32AppServer::Init(AppDelegate * sAppDelegate)
     }
     chip::Server::GetInstance().Init(initParams);
 
-    // Initialize device attestation config
-#if CONFIG_ENABLE_ESP32_FACTORY_DATA_PROVIDER
-    SetDeviceAttestationCredentialsProvider(&sFactoryDataProvider);
```

Comment:
Removing this function from `AppServer::Init` could cause commissioning issue for other example since all of the ESP examples use `AppServer::Init` to initialize chip server.

Reply:
Okay, so I replicated the change in lighting-app in rest of the examples